src/Features/CSharp/Portable/CodeRefactorings/ConvertToRecord/CSharpConvertToRecordRefactoringProvider.cs

```diff
@@ -0,0 +1,623 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Linq;
```

Comment:
interesting!

Reply:
Odd, I think my unused imports checker is broken because I can delete them and everything still works. I think they got in there because I was accidentally Intelisense completing `XmlNode` when trying to type `XmlNodeSyntax`